packages/help-center/src/hooks/use-get-history-chats.ts

```diff
@@ -58,7 +58,20 @@ export const useGetHistoryChats = (): UseGetHistoryChatsResult => {
 				allConversations,
 				allSupportInteractions
 			);
-			setConversations( filteredConversations );
+
+			const conversationsWithUpdatedStatuses = filteredConversations.map( ( conversation ) => {
+				const supportInteraction = allSupportInteractions.find(
+					( interaction ) => interaction.uuid === conversation.metadata?.supportInteractionId
+				);
+
+				if ( supportInteraction ) {
+					conversation.metadata.status = supportInteraction.status;
```

Comment:
Aren't we still mutating the `conversation` objects that way? Am I perhaps missing a place earlier where we create new objects?

Reply:
Yes you're right, I missed the logic here. I update the PR. A new conversation object is created and mutated as I `map` through the `filteredConversations` array.